File: hang.liu/832.js
var flipAndInvertImage = function(A) {
    var result = []
    
    for (var item of A) {
        var temp = []
        
        for (var num of item) {
            temp.unshift(1 - num)
        }
        
        result.push(temp)
        
    } 
    
    return result
};

File: xiang.li/832.java
class Solution {
    public int[][] flipAndInvertImage(int[][] A) {
        for(int i = 0;i < A.length;i++) {
            for(int js = 0, je = A[0].length - 1;je >= js;js++, je--) {
                if(A[i][js] == A[i][je]) {
                    A[i][js] = A[i][je] = A[i][je] == 0 ? 1: 0;
                }
            }
        }
        return A;
    }
}
